roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -37,6 +37,45 @@ spec:
 {% endfor %}
 {% endif %}
       containers:
+        - name: oauth-proxy
+          image: {{openshift_logging_elasticsearch_oauth_proxy_image_prefix}}:{{openshift_logging_elasticsearch_oauth_proxy_image_version}}
+          imagePullPolicy: Always
+          args:
+          - >-
+            cat /etc/elasticsearch/secret/admin-ca >> /etc/pki/tls/certs/ca-bundle.crt && /bin/oauth-proxy
+            --https-address=:4443
+            -provider=openshift
+            -client-id=system:serviceaccount:{{openshift_logging_elasticsearch_namespace}}:aggregated-logging-elasticsearch
+            -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
+            -cookie-secret={{ 16 | oo_random_word | b64encode }}
+            -upstream=https://localhost:9200
+            '-openshift-sar={"namespace": "{{openshift_logging_elasticsearch_namespace}}", "verb": "get", "resource": "services"}'
```

Comment:
Can we change the "services" resource to something specific like "prometheus-logs" ?